apps/entry/mutations.py

```diff
@@ -322,7 +322,7 @@ def _get_notification_type(event):
                 event=instance.event,
             )
 
-        Figure.update_event_status_and_send_notifications(instance.event)
+        Figure.update_event_status_and_send_notifications(instance.event.id)
```

Comment:
`event.id` will fetch the event.

```suggestion
        Figure.update_event_status_and_send_notifications(instance.event_id)
```

Reply:
It's better to pass the event directly? We may need to discuss about this.